integration

```diff
@@ -714,6 +714,7 @@
   "PeteRager/lennoxs30",
   "petergridge/irrigation-V5",
   "petergridge/openweathermaphistory",
+  "peterturza/home-assistant-jablotron-futura",
```

Comment:
The repository contains virtually no documentation, it also looks abandoned already?